_includes/releases/v23.1/v23.1.0-beta.3.md

```diff
@@ -0,0 +1,63 @@
+
+<h3 id="v23-1-0-beta-3-sql-language-changes">SQL language changes</h3>
+
+- Statements of type [`SET ...`](../v23.1/set-vars.html) are no longer displayed on the [Insights page](../v23.1/ui-insights-page.html). [#101672][#101672]
+
+<h3 id="v23-1-0-beta-3-operational-changes">Operational changes</h3>
+
+- The amount of [replication](../v23.1/architecture/replication-layer.html) traffic in flight from a single [Raft leader](../v23.1/architecture/reads-and-writes-overview.html#architecture-raft-leader) to a follower has been reduced from 256 MB to 32 MB. This reduces the chance of running out of memory during bulk write operations. This can be controlled via the environment variable `COCKROACH_RAFT_MAX_INFLIGHT_BYTES`. [#101508][#101508]
+
+<h3 id="v23-1-0-beta-3-db-console-changes">DB Console changes</h3>
+
+- When a user activates diagnostics on the [SQL Activity](../v23.1/ui-statements-page.html#statement-fingerprints-view) page, there is now an option to select the trace rate for statement diagnostics collection. [#101762][#101762]
+- Updated the **Network Latency** side navigation name and Network Diagnostics page title to Network. Updated the Advanced Debugging page title to [Advanced Debug](../v23.1/ui-debug-pages.html). [#101761][#101761]
+- Introduced a "draining node" **Node Status** on the [Cluster Overview panel](../v23.1/ui-cluster-overview-page.html#cluster-overview-panel) so that "draining node" has it own value instead of counting as a "dead node". [#101794][#101794]
+- Added a time scale selector to the [**Diagnostics** tab](../v23.1/ui-statements-page.html#diagnostics) under the Statement Details page. This allows users to view bundles from the selected period only. [#101803][#101803]
+
+<h3 id="v23-1-0-beta-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that caused internal errors when executing [user-defined functions](../v23.1/user-defined-functions.html) with empty bodies. This bug was only present in alpha pre-release versions of 23.1. [#101383][#101383]
+- Fixed a bug that caused a [restore](../v23.1/restore.html) to fail occasionally due to incorrect schema ID resolution when restoring a [backup](../v23.1/backup.html) with user-defined schemas. [#101310][#101310]
+- Fixed a bug in parsing a [`search_path`](../v23.1/sql-name-resolution.html#current-schema) with a quote in it when specified in the connection string. Also, the `search_path` session variable now supports schema names that have commas in them. [#101492][#101492]
+- Fixed a bug that has existed since [user-defined functions](../v23.1/user-defined-functions.html) were introduced that could cause a function call to resolve to the wrong function after changes to the schema search path. [#101513][#101513]
```

Comment:
'schema search path' could link to https://www.cockroachlabs.com/docs/v23.1/sql-name-resolution#current-schema (as I see you have elsewhere)